src/js/wand.js

```diff
@@ -1,24 +1,31 @@
-(function() {
-  Wand = function(options) {
+var Wand = (function() {
+
+  var wand = {};
+
+  var options = {},
+      nodes = [],
+      elem = null;
+
+  var badHtmlError = new Error('You did not pass a valid HTML Element');
+
+  wand.init = function(opts) {
```

Comment:
I agree that would be nice. @creade and I went in a lot of circles this afternoon, and I can't remember if we nixed the idea for a reason or because we just didn't get that far. Maybe he can remember? @creade?